src/scheduler/fifo.c

```diff
@@ -569,8 +569,8 @@ schedule(int cmd, int sd, char *runjobid)
 			 * This is required since there is a probability that scheduler's configuration has been changed at
 			 * server through qmgr.
 			 */
-			if (!update_svr_schedobj(connector, 0, 0)) {
-				log_err(-1, __func__, "update_svr_schedobj failed");
+			if (!validate_sched_attrs(connector)) {
+				log_err(-1, __func__, "validate_sched_attrs failed");
```

Comment:
The failure message should be in this function itself.  You shouldn't log here.  If you are logging in the function, then you'll get 2 messages.

Reply:
I have now removed the logging inside the function validate_sched_attrs.